src/app/user-settings/settings/page.tsx

```diff
@@ -1,3 +1,269 @@
+    </div>
+  );
+}
+
+function ToggleSwitch({ id }: { id: string }) {
+  const [isChecked, setIsChecked] = useState(true);
+  return (
+    <label
+      htmlFor={id}
+      className={clsx({
+        "relative h-10 w-20 rounded-full": true,
+        "bg-[#A6A6A6]": isChecked,
+        "bg-[#69A297]": !isChecked,
+      })}
+    >
+      <input
+        type="checkbox"
+        id={id}
+        className="peer sr-only"
+        onChange={(e) => setIsChecked(e.target.checked ? false : true)}
+      />
+      <span className="absolute left-11 top-1 h-4/5 w-2/5 rounded-full bg-white drop-shadow-lg transition-all duration-300 peer-checked:left-1"></span>
+      <span className="absolute left-2.5 top-2 h-4/5 w-2/5 text-base text-white transition-all duration-300 peer-checked:opacity-0">
+        OFF
+      </span>
+      <span className="absolute right-3.5 top-2 text-base text-white opacity-0 transition-all duration-300 peer-checked:opacity-100">
+        ON
+      </span>
+    </label>
+  );
+}
+
 export default function Settings() {
-  return <div className="md:pl-56">Settings Page</div>;
+  return (
+    <div className="relative flex grow flex-col gap-8 py-6 md:pl-56">
+      {/* Feature Section */}
+      <section>
+        <div className="flex items-center gap-1">
+          <h3 className="text-xl font-semibold">FEATURES</h3>
+          <FeatureIcon className="h-7 w-7" />
+        </div>
+        <p className="mb-4 text-lg text-stone-500">
+          Toggle article features on and off
+        </p>
+        <ul className="divide-y rounded-md border bg-white px-4 text-xl font-semibold">
+          <li className="flex h-16 items-center justify-between">
+            Reading History
+            <ToggleSwitch id="reading-history" />
+          </li>
+          <li className="flex h-16 items-center justify-between">
+            Show quizzes on articles
+            <ToggleSwitch id="show-quizzes" />
+          </li>
+          <li className="flex h-16 items-center justify-between">
+            Show in-line comments
+            <ToggleSwitch id="show-comments" />
+          </li>
+        </ul>
+      </section>
+
+      {/* Interests Section */}
+      <section>
+        <div className="flex items-center gap-1">
+          <h3 className="text-xl font-semibold">INTERESTS</h3>
+          <InterestIcon className="h-7 w-7" />
+        </div>
+        <p className="mb-4 text-lg text-stone-500">
+          Pick the curated content you want to see most.
+        </p>
+        <div className="grid max-h-64 auto-rows-max grid-cols-1 gap-2 overflow-auto rounded-md border bg-white p-4 md:grid-cols-2">
+          <div className="flex items-center gap-1">
+            <input type="checkbox" value="Chemistry" className="mr-2" />
+            <ChemistryIcon className="w-5" />
+            Chemistry
+          </div>
+          <div className="flex items-center gap-1">
+            <input type="checkbox" value="Chemistry" className="mr-2" />
```

Comment:
Are all of these supposed to be Chemistry?

Reply:
This is a mistake. Forgot to change the values. lol